### ISSUE TITLE
Add bin icon to delete car in my cars scree 

### ISSUE THREAD

**Issue by neel1998** [enviroCar/enviroCar-app#448]
Currenty the delete option appears when user clicks on the car. Which is not very much apparent for the first time user. Having a bin icon would make it easy

**Current behavior**
![20200221_022457](https://user-images.githubusercontent.com/32041242/74977954-b7be2f00-5451-11ea-9071-ca8258e9a64c.gif)

**I would like to work on this issue**

**Comment by asaikarthikeya** [comment]
Working on this.

**Comment by cdhiraj40** [comment]
@asaikarthikeya  what exactly mean by "having a bin icon" ? adding it around the readiobutton?

**Comment by asaikarthikeya** [comment]
> @asaikarthikeya  what exactly mean by "having a bin icon" ? adding it around the radiobutton?
> if the issue/feature is still relevant I would like to try :)

If you see OBD selection screen, you can see that for every paired device it has a bin symbol(to delete)and  a select option. Similar thing is expected in car selection screen

Let's wait to hear @SebaDro views on this issue. If sir feels this feature is good to add, you can work on this. Thanks 🙂

**Comment by cdhiraj40** [comment]
> > @asaikarthikeya  what exactly mean by "having a bin icon" ? adding it around the radiobutton?
> > if the issue/feature is still relevant I would like to try :)
> 
> If you see OBD selection screen, you can see that for every paired device it has a bin symbol(to delete)and a select option. Similar thing is expected in car selection screen
> 
> Let's wait to hear @SebaDro views on this issue. If sir feels this feature is good to add, you can work on this. Thanks

sure I will wait :)

**Comment by asaikarthikeya** [comment]
Definitely, bin icon provides a better ui experience but not really sure if this is required in our app because, some car models have really long name, such car names are not displayed completely.  This is due to the presence of car symbol on the left side. By adding a bin symbol on the right end the car name will be further be reduced in size. May be only the car manufacturer name will be visible. In my opinion this is may not suit in our app ui.

@SebaDro is this required sir? What are your thoughts on this? Thanks

**Comment by cdhiraj40** [comment]
yes, I totally agree if we don't show car names in the car list then I guess it doesn't make sense.
if @SebaDro would like this feature then I will sure try to work on it else I will try doing something else. Thanks

> Definitely, bin icon provides a better ui experience but not really sure if this is required in our app because, some car models have really long name, such car names are not displayed completely. This is due to the presence of car symbol on the left side. By adding a bin symbol on the right end the car name will be further be reduced in size. May be only the car manufacturer name will be visible. In my opinion this is may not suit in our app ui.
> 
> @SebaDro is this required sir? What are your thoughts on this? Thanks

yes, I totally agree if we don't show car names properly in the car list then I guess it doesn't make sense.
if @SebaDro would like this feature then I will sure try to work on it else I will try doing something else. Thanks

**Comment by SebaDro** [comment]
The current workflow for deleting an entry is quite sufficient. An extra bin icon would be only nice to have, so priority for this enhancement is very low. However, if you come up with a good design, that also takes into account long device names, I would consider to merge it.

**Comment by pree-T** [comment]
May I work on it?

**Comment by SanjaySargam** [comment]
Can I work on this issue?
